jekyll/assets/css/main.scss

```diff
@@ -576,7 +576,7 @@ span.tag {
   margin-top: 15pt;
   margin-bottom: 15pt;
 
-  h1 {
+  h1, h2 {
```

Comment:
This most likely affects other h2 headings on other pages. You need to fix this a different way.